### PR TITLE
rock 0.9.10

### PR DESCRIPTION
**Pull request by fasterthanlime**
Notes:

  * I've changed my username on github, I used to be @nddrylliog — git URLs have been adjusted in consequence (although github's redirect works well if you want to verify I'm me)
  * Thanks @BrewBot for adding bottles, but I'm not sure what I'm supposed to do with them in a PR for a new version? I supposed it'll just re-add them when they're ready?

Let me know if I messed up anything & see you next year for 0.9.11!

### CODE REVIEW

File: Library/Formula/rock.rb [hunk 1]
```diff
@@ -3,18 +3,10 @@
 class Rock < Formula
   desc "ooc compiler written in ooc"
   homepage 'http://ooc-lang.org'
-  url 'https://github.com/nddrylliog/rock/archive/v0.9.9.tar.gz'
-  sha1 '11d4a46320e2b538989354505e0a6ac4311b049f'
+  url 'https://github.com/fasterthanlime/rock/archive/v0.9.10.tar.gz'
+  sha1 'c3bc1847d7e1753c66d04a2b333cbb405053d46f'
```
Comment (DomT4) [inline Library/Formula/rock.rb]: You'll need to use `sha256` here. We've deprecated sha1.

File: Library/Formula/rock.rb [hunk 2]
```diff
@@ -3,18 +3,10 @@
...
 
-  head 'https://github.com/nddrylliog/rock.git'
-
-  bottle do
```
Comment (DomT4) [inline Library/Formula/rock.rb]: Leave this block intact please. We'll update it with new bottles automatically when we pull.

**Comment by fasterthanlime** [comment]
@DomT4 force-pushed a squashed commit that contains SHA-256 hashes & doesn't touch the bottle block.

**Comment by bfontaine** [comment]
Thank you for your contribution to Homebrew, @fasterthanlime! :tada:

**Pull request closed without merging**